src/test/java/com/google/cloud/solutions/autotokenize/common/TransformingReaderTest.java

```diff
@@ -84,23 +85,19 @@ public JdbcReaderTest(
       this.testDatabaseName = "test_" + UUID.randomUUID().toString().replaceAll("[\\-]", "_");
       this.secretsStub =
           ConstantSecretVersionValueManagerServicesStub.of("id/to/secrets/key/version", "");
-    }
 
-    @BeforeClass
-    public static void setUpTestDbInstance() throws Exception {
-      testDatabase = DB.newEmbeddedDB(0);
-      testDatabase.start();
+      databaseContainer = 
+        new MySQLContainer(DockerImageName.parse("mysql:8.0.24"))
+          .withDatabaseName(testDatabaseName);
```

Comment:
FYI `databaseContainer` is a static field, so setting `withDatabaseName` on it would only make sense once (before the first `start()` call), and other attempts will be ignored (since the container is already started)

I would highly recommend either making `testDatabaseName` static as well or maybe not randomizing the DB name at all, since containers are ephemeral and always start fresh

The same applies to `initScriptFile`

Reply:
@anantdamle mentioned that this will be refactored later.